research-hub-web/src/app/components/search-page/search-page.component.html

```diff
@@ -17,57 +17,75 @@
-></app-search-filter-bar>
+<mat-tab-group mat-align-tabs="center">
 
-<!-- Search Results -->
-<div
-  id="search-results"
-  ngClass.gt-sm="search-content site-padding-2 section"
-  ngClass.lt-md="site-padding search-content section"
->
-  <app-search-results-list
-    [searchResults]="{ results: searchResults, totalResults: totalResults }"
-    [loading]="loading"
-  ></app-search-results-list>
-  <!--Load more or no results message-->
-  <div
-    class="eol"
-    *ngIf="
-      !loading && searchResults.length === totalResults && totalResults !== 0
-    "
-  >
-    <div>No More Results</div>
-  </div>
+  <mat-tab label="ResearchHub">
+    <app-search-filter-bar
+      [searchText]="searchText"
+      [totalResults]="totalResults"
+      [(sortOrder)]="sortOrder"
+      [(activeFilters)]="activeFilters"
+    ></app-search-filter-bar>
 
-  <!-- No results message -->
-  <div *ngIf="totalResults === 0" class="no-results">
-    <p>
-      Sorry, your search for "<i
-        ><b>{{ searchText }}</b></i
-      >" did not match anything on the ResearchHub.
-    </p>
-    <p>
-      <br /><b>Suggestions:</b><br />
-      Widen your filters or modify your search query<br />
-      Let us know what you were looking for via our
-      <a href="{{ feedbackUrl }}">Feedback page</a><br />
-      You can try the Staff Service Centre in the
-      <a href="{{ staffIntranet }}">Staff Intranet</a><br />
-    </p>
-  </div>
-</div>
+    <!-- Search Results -->
+    <div
+      id="search-results"
+      ngClass.gt-sm="search-content site-padding-2 section"
+      ngClass.lt-md="site-padding search-content section"
+    >
+      <app-search-results-list
+        [searchResults]="{ results: searchResults, totalResults: totalResults }"
+        [loading]="loading"
+      ></app-search-results-list>
+      <!--Load more or no results message-->
+      <div
+        class="eol"
+        *ngIf="
+          !loading && searchResults.length === totalResults && totalResults !== 0
+        "
+      >
+        <div>No More Results</div>
+      </div>
+
+      <!-- No results message -->
+      <div *ngIf="totalResults === 0" class="no-results">
+        <p>
+          Sorry, your search for "<i
+            ><b>{{ searchText }}</b></i
+          >" did not match anything on the ResearchHub.
+        </p>
+        <p>
+          <br /><b>Suggestions:</b><br />
+          Widen your filters or modify your search query<br />
+          Let us know what you were looking for via our
+          <a href="{{ feedbackUrl }}">Feedback page</a><br />
+          You can try the Staff Service Centre in the
+          <a href="{{ staffIntranet }}">Staff Intranet</a><br />
+        </p>
+      </div>
+    </div>
+
+    <!-- Floating up button -->
+    <button
+      mat-mini-fab
+      matTooltip="Scroll to top"
+      class="floating-button"
+      (click)="scrollToTop()"
+      >
+      <mat-icon>arrow_drop_up</mat-icon>
+    </button>
+  </mat-tab>
```

Comment:
See [48b26b97](https://github.com/UoA-eResearch/hub-stack/pull/222/commits/48b26b9790404892020c152793e39846da1143ef). SPlit into separate component, and also created NoResultsComponent.

The floating up button is kept in the search page now rather than duplicating it in the two tabs ([f72c448](https://github.com/UoA-eResearch/hub-stack/pull/222/commits/f72c448f869ffbd227d1625ed5a05738906586fb))